sdk/examples/client/07_mqtt.rs

```diff
@@ -1,22 +1,35 @@
 // Copyright 2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-//! TODO: Example description
+//! This example shows how to listen to MQTT events of a node.
 //!
-//! cargo run --example 07_mqtt --features=mqtt --release
+//! Rename `.env.example` to `.env` first, then run the command:
+//! ```sh
+//! cargo run --release --all-features --example 07_mqtt [NUM_EVENTS] [ADDRESS]
+//! ```
 
-use iota_sdk::client::{
-    mqtt::{MqttEvent, MqttPayload, Topic},
-    Client, Result,
+use iota_sdk::{
+    client::{
+        mqtt::{BrokerOptions, MqttEvent, MqttPayload, Topic},
+        Client, Result,
+    },
+    types::block::address::Bech32Address,
 };
 
 // Connecting to a MQTT broker using raw ip doesn't work with TCP. This is a limitation of rustls.
 #[tokio::main]
 async fn main() -> Result<()> {
-    // Create a client instance
+    let num_events: usize = std::env::args().nth(1).map(|s| s.parse().unwrap()).unwrap_or(10);
```

Comment:
NIT. Maybe the same like in other examples:
`let mut args = std::env::args().skip(1);` and then `next`?

Reply:
actually, no. I want to use `nth(x)` wherever possible because it's more convenient. I'll change those examples that don't do that so it's consistent.

Reply:
Done